### ISSUE TITLE
[FEATURE] write company name after logo also hover effect in navlinks

### ISSUE THREAD

**Issue by Iamdivyak** [tamalCodes/Milan#744]
### Description

It will be a good idea if the name of the company (MILAAN) is available beside the logo in the nav bar with a good font like Montserrat

And some hover effect in nav links for better user Experience

Kindly assign this task to me under GSSOC contributor

### Screenshots

<img width="960" alt="image" src="https://github.com/IAmTamal/Milan/assets/102896170/d42baa9d-5177-4616-bfde-30c703ce1c77">
 

### 🥦 Browser

Google Chrome

### Checklist ✅

- [X] I checked and didn't find similar issue
- [X] I have read the [Contributing Guidelines](https://github.com/IAmTamal/Milan/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (blank for no)

**Comment by IAmTamal** [comment]
Great idea, but since #740 is working on adding a new button make sure you don't do a lot of changes - plus there might be chances of conflicts. So for now I will put this on hold, and after #740  is completed we will move forward.
